hw/lint/tools/veriblelint/parse-lint-report.py

```diff
@@ -70,10 +70,8 @@ def get_results(resdir):
         # this is a workaround until we actually have native Edalize
         # support for JasperGold and "formal" targets
         ("warnings",
-         r"^(?!WARNING: Unknown item formal in section Target)WARNING: .*"
-         # TODO(https://github.com/lowRISC/ibex/issues/1033):
-         # remove once this has been fixed in Edalize or in the corefile.
-         r"^(?!WARNING: Unknown item symbiyosis in section Target)WARNING: .*"
+         r"^(?!WARNING: Unknown item formal in section Target)"
+         r"WARNING: .*"
```

Comment:
this basically repeats what we've done on this (reverted commit): https://github.com/lowRISC/opentitan/commit/9ce4bd9eba8ee99a301ce52f136ca70aa06ccf93